system-test/stake-operations-testcases/offline_stake_operations.sh

```diff
@@ -4,13 +4,26 @@
 # shellcheck disable=SC2068
 # shellcheck disable=SC2206
 # shellcheck disable=SC2162
+# shellcheck disable=SC2178
+# shellcheck disable=SC2145
 
 # shellcheck disable=SC1090
 # shellcheck disable=SC1091
 source "$(dirname "$0")"/../automation_utils.sh
 
 set -e
 
+function get_signers_string() {
+  sign_only_output="$1"
+
+  signers=()
+  while read LINE; do
+    signers+=( --signer $LINE )
+  done <<<"$(sed -Ee $'s/^  ([a-zA-Z1-9][^OIl]+=[a-zA-Z1-9][^OIl]+)/\\1/\nt\nd' <<<"$sign_only_output")"
```

Comment:
This regex doesn't quite do what we want
```
$ sed -Ee $'s/^  ([a-zA-Z1-9][^OIl]+=[a-zA-Z1-9][^OIl]+)/\\1/\nt\nd' <<<'  A~~~~~=B!!!!!'
A~~~~~=B!!!!!
```

Try this
```suggestion
  done <<<"$(sed -Ee $'s/^  ([a-km-zA-HJ-NP-Z1-9]{32,44}=[a-km-zA-HJ-NP-Z1-9]{64,88})$/\\1/\nt\nd' <<<"$sign_only_output")"
```

Reply:
done